jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -403,6 +404,11 @@ def state_attr_for_line_visible(state):
                                                  state_filter=is_image)
 
         self.stretch_histogram = Plot(self, viewer_type='histogram')
+        # Add the stretch bounds tool to the default Plot viewer.
+        self.stretch_histogram.tools_nested.append(["jdaviz:stretch_bounds"])
+        self.stretch_histogram.toolbar = NestedJupyterToolbar(self.stretch_histogram.viewer,
+                                                              self.stretch_histogram.tools_nested,
+                                                              [])

```

Comment:
Might as well make this the default tool so its on by default!

```suggestion
                                                              ["jdaviz:stretch_bounds"])

```

Reply:
Good call!